package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmsgov/qpp-measures-data",
-  "version": "0.0.21",
+  "version": "0.0.22",
```

Comment:
ach, I didn't realize we were publishing this privately (since it's open source), explains some issues I had with yarn work!